internal/vpn/os_linux.go

```diff
@@ -0,0 +1,68 @@
+//+build linux
+
+package vpn
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+)
+
+const (
+	gatewayForIfcCMDFmt         = "/sbin/route -n | /bin/grep %s | /usr/bin/awk '$1 == \"0.0.0.0\" {print $2}'"
```

Comment:
Seems the command `route` is deprecated: https://unix.stackexchange.com/questions/83096/what-is-the-difference-between-sbin-ip-route-and-sbin-route

For example, on Manjaro, this command does not exist.

Also, not all distros put binaries in `/sbin`. Would be be able to do: `$(command -v ip)` (assuming we are to use the `ip` command).

Reply:
@evanlinjin we still have to maintain such commands for Mac systems at least. directory with the binary is actually a problem, since `PATH` seems to be lost when we try to execute command from the process which is itself a command executed by another process. I'll try to deal with this within this PR. but regarding moving to `ip` command set for Linux systems, I'll put this point into https://github.com/SkycoinProject/skywire-mainnet/issues/347 , I think it's best to be done there

Reply:
> @evanlinjin we still have to maintain such commands for Mac systems at least. directory with the binary is actually a problem, since `PATH` seems to be lost when we try to execute command from the process which is itself a command executed by another process. I'll try to deal with this within this PR. but regarding moving to `ip` command set for Linux systems, I'll put this point into #347 , I think it's best to be done there

@Darkren I thought `command` was a shell built-in? And sure, handling the `ip` situation elsewhere is fine.

Reply:
@evanlinjin tried running `$(command -v ifconfig)` got PATH error. so, just passed correct PATH to the app cmd, seems to work fine with just the binary name